pddl/logic/predicates.py

```diff
@@ -127,7 +127,7 @@ def __hash__(self) -> int:
 
     def __str__(self) -> str:
         """Get the string representation."""
-        return f"({Symbols.EQUAL} {self.left} {self.right})"
+        return f"({Symbols.EQUAL.value} {self.left} {self.right})"
```

Comment:
No chance of getting a test in for this?

Reply:
Kinda surprised we didn't have it crop up before :P

Reply:
Yep! It was a bit surprising also to me.